stable/vulcan/values.yaml

```diff
@@ -219,6 +219,15 @@ scanengine:
     snsArn: arn:aws:sns:local:012345678900:VulcanK8SScans
     snsChecks:
       topicArn: arn:aws:sns:eu-west-1:012345678900:VulcanK8SScanEngineChecks
+  configFiles:
+    queues.yaml: |
+      queues:
+      - name: default
+        arn: arn:aws:sqs:local:012345678900:VulcanK8SChecksGeneric
+      - name: nessus
+        arn: arn:aws:sqs:local:012345678900:VulcanK8SChecksTenable
+        checks:
```

Comment:
I think this should be _checktypes_, at least according to latest scan engine implementation:
https://github.mpi-internal.com/spt-security/vulcan-scan-engine/blob/a78e6a0952f85689b7dda244a688f2ef7782e02a/cmd/vulcan-scan-engine/commands/root.go#L172